doc/GettingStarted.md

```diff
@@ -122,6 +125,64 @@ In addition, Vitess requires the software and libraries listed below.
     ``` sh
     $ sudo apt-get install openjdk-7-jre
     ```
+    
+#### OS X
+
+1.  [Install Homebrew](http://brew.sh/).
+
+2.  On OS X, MySQL 5.6 has to be used, MariaDB doesn't work for some reason yet. It should be installed from Homebrew
+    (install steps are below).
+   
+3.  ZooKeeper is used as lock service.
+
+4.  Run the following commands:
+
+    ``` sh
+    brew install go automake libtool memcached python mercurial git bison curl wget homebrew/versions/mysql56
+    pip install --upgrade pip setuptools
+    pip install virtualenv
+    pip install MySQL-python
+    ```
+    
+5.  Install Java runtime from this URL: https://support.apple.com/kb/dl1572?locale=en_US
```

Comment:
I tried to install Java 8 JRE at first, but when I was trying to run the java command from shell, I always got a window saying that I need to install Java and both of the buttons in the dialog did nothing, so I needed to install the old Java to make it work. But I'm not sure which version is being used by ZooKeeper in the runtime, I will have to check that, since there are both Java version installed in the VM.

I'll have a look at that later in the evening.